soc/arm/nuvoton_npcx/common/soc_miwu.h

```diff
@@ -139,6 +139,23 @@ void npcx_miwu_irq_disable(const struct npcx_wui *wui);
  */
 bool npcx_miwu_irq_get_state(const struct npcx_wui *wui);
 
+/**
+ * @brief Get interrupt pending bit of the wake-up input source
+ *
+ * @param wui A pointer on wake-up input source
+ *
+ * @retval 1 if interrupt is pending
+ */
+bool npcx_miwu_irq_get_pending(const struct npcx_wui *wui);
+
+/**
+ * @brief Clear interrupt pending bit of the wake-up input source
+ *
+ * @param wui A pointer on wake-up input source
+ *
+ */
+void npcx_miwu_irq_clear_pending(const struct npcx_wui *wui);
```

Comment:
Do we really want these separate, or would get_and_clear() be useful as a combined function?

Reply:
Done.